Documentation/networking.md

```diff
@@ -1,37 +1,54 @@
 # Networking
 
-`rkt` has two networking options: "host" networking where the apps runs in the root host network namespace or "private" where the apps are allocated a new IP address and a new network namespace.
+On some of rkt's subcommands *([run](subcommands/run.md), [run-prepared](subcommands/run-prepared.md))*, the `--net` flag allows you to configure the pods network.
+The various options can be grouped by two categories:
 
-## Host (default) mode
+* [host mode](#host mode)
+* [contained mode (default)](#contained mode)
 
-By default, `rkt run` will start the pod with host networking.
-This means that the apps within the pod will share the network stack and the interfaces with the host machine.
+## Host mode
+When `--net=host` is passed the pod's apps will inherit the network namespace its caller.
 
-## Private networking mode
+If rkt is directly called from the host the apps within the pod will share the network stack and the interfaces with the host machine.
+This means that every network service that runs in the pod has the same connectivity as if it was started on the host directly.
+
+## Contained mode
+If anything other than `host` is passed to `--net=`, the pod will live in a separate network namespace with the help of [CNI](https://github.com/appc/cni) and it's plugin system.
+The network setup for the pod's network namespace depends on the available CNI configuration files that are shipped with rkt and also configured by the user.
+
+### Network selection
+Every network must have a unique name and can only be joined once by every pod.
+Passing a list of comma separated network as in `--net=net1,net2,net3,...` tells rkt which networks should be joined.
+This is useful for grouping certain pods networks together while separating others.
+There is also the possibility to load all configured networks by using  `--net=all`.
+
+### Builtin networks
+rkt ships with two builtin networks that are named *default* and *default-restriced*.
 
-If `rkt run` is started with the `--private-net` flag, the pod will be executed with its own network stack, with the default network plus all configured networks.
-This is equivalent to passing `--private-net=all`.
-Passing a list of comma separated network names as in `--private-net=net1,net2,net3,...` restricts the network stack to the specified networks.
-This can be useful for grouping certain pods together while separating others.
 
 ### The default network
-The default network consists of a loopback device and a veth device.
+The *default* network is loaded automatically in three cases: `--net` is not present on the command line and if `--net` or `--net=default` is passed.
```

Comment:
can you just put these as a list?
- `--net` is not passed on the command-line
- `--net` is passed with no parameters
- `--net=default` is passed